src/Services.java

```diff
@@ -0,0 +1,236 @@
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.time.LocalDate;
+import java.util.Scanner;
+
+public class Services {
+
+    public static final String WHITE_UNDERLINED = "\033[4;37m";
+    public static final String ANSI_GREEN = "\u001B[32m";
+    public static final String ANSI_RESET = "\u001B[0m";
+
+    final String currency = "RON";
+
+    String service,destination;
+    private CheckInput check = new CheckInput();
+    private HistorySaver saveToFile = new HistorySaver();
+    private Scanner scanner = new Scanner(System.in);
+
+    LocalDate date;
```

Comment:
i would use an international date, not a local one
just to be sure there will be no timezone problems

Reply:
I am using only the date, but not the hours.